src/util.rs

```diff
@@ -77,8 +77,8 @@ mod tests {
 
     #[test]
     fn test_into_owned() {
```

Comment:
It tests that `into_owned()` on strings is the identity operation :) I should probably have added also `"abcde".into_owned()` as well.